.changeset/friendly-oranges-rescue.md

```diff
@@ -0,0 +1,5 @@
+---
+"@enzymefinance/sdk": patch
+---
+
+added redeem shares in kind method and test
```

Comment:
Can you uppercase the first letter here and write these as full sentences (with punctuation etc.)? The text here ends up in the changelogs.